tests/Unit/Domain/Creators/Test_BinaryCompactObject.cpp

```diff
@@ -565,6 +565,71 @@ void test_bbh_time_dependent_factory(const bool with_boundary_conditions,
   const std::vector<double> times_to_check{{1., 2.3}};
   const auto domain = TestHelpers::domain::creators::test_domain_creator(
       *binary_compact_object, with_boundary_conditions, false, times_to_check);
+
+  const auto& blocks = domain.blocks();
+  const auto& final_block = blocks[blocks.size() - 1];
+
+  std::unordered_map<std::string, ExcisionSphere<3>>
+      expected_excision_spheres{};
+  expected_excision_spheres.emplace(
+      "ExcisionSphereA",
+      ExcisionSphere<3>{1.0,
+                        tnsr::I<double, 3, Frame::Grid>{{3.0, 0.0, 0.0}},
+                        {{0, Direction<3>::lower_zeta()},
+                         {1, Direction<3>::lower_zeta()},
+                         {2, Direction<3>::lower_zeta()},
+                         {3, Direction<3>::lower_zeta()},
+                         {4, Direction<3>::lower_zeta()},
+                         {5, Direction<3>::lower_zeta()}}});
+  if (with_time_dependence) {
+    expected_excision_spheres.at("ExcisionSphereA")
+        .inject_time_dependent_maps(
+            final_block.moving_mesh_grid_to_inertial_map().get_clone());
+  }
+  if (excise_B) {
+    expected_excision_spheres.emplace(
+        "ExcisionSphereB",
+        ExcisionSphere<3>{0.2,
+                          tnsr::I<double, 3, Frame::Grid>{{-2.0, 0.0, 0.0}},
+                          {{12, Direction<3>::lower_zeta()},
+                           {13, Direction<3>::lower_zeta()},
+                           {14, Direction<3>::lower_zeta()},
+                           {15, Direction<3>::lower_zeta()},
+                           {16, Direction<3>::lower_zeta()},
+                           {17, Direction<3>::lower_zeta()}}});
+    if (with_time_dependence) {
+      expected_excision_spheres.at("ExcisionSphereB")
+          .inject_time_dependent_maps(
+              final_block.moving_mesh_grid_to_inertial_map().get_clone());
+    }
+  }
+
+  const auto& excision_spheres = domain.excision_spheres();
+  CHECK(excision_spheres == expected_excision_spheres);
+
+  const auto check_excision_sphere_map =
+      [&binary_compact_object](const ExcisionSphere<3>& excision_sphere) {
+        const double time = 1.0;
+        // Taken from option string above
+        const auto functions_of_time =
+            binary_compact_object->functions_of_time();
+        const auto& center = excision_sphere.center();
+        const auto& map = excision_sphere.moving_mesh_grid_to_inertial_map();
+
+        const auto mapped_point = map(center, time, functions_of_time);
+
+        // Should be same point
+        for (size_t i = 0; i < 3; i++) {
+          CHECK(center.get(i) == approx(mapped_point.get(i)));
```

Comment:
This still doesn't really check the map. While you're rebasing, maybe add the quick rotation check like you have in the cylindrical domain?

Reply:
I can't really do that because the cylindrical domain had the expansion map turned off so it made the map easy (just a rotation). This one has the expansion map so the mapped point is more complicated and I'd rather not reimplement the map here, or make a lot of edits to turn off the expansion map for the whole test.

Also I don't really see the purpose of testing the map? The maps are already tested elsewhere. What we really care about is that the map itself is the map we expect which is tested above when I check the expected excision spheres against the ones created in the domain creator.

Reply:
ok